### ISSUE TITLE
Pytest warnings about unknown marks

### ISSUE THREAD

**Issue by emersonknapp** [ament/ament_lint#145]
Pytest for the linters has started emitting `PytestUnknownMarkWarning`s

I've started seeing these warnings in my development environment as well. Based on the error output, it seems that we can register the linter marks and stop these warnings.

Sample output from https://ci.ros2.org/job/ci_osx/6068/console 

```
17:29:00 =============================== warnings summary ===============================
17:29:00 /Users/osrf/jenkins-agent/workspace/ci_osx/venv/lib/python3.7/site-packages/_pytest/mark/structures.py:337
17:29:00   /Users/osrf/jenkins-agent/workspace/ci_osx/venv/lib/python3.7/site-packages/_pytest/mark/structures.py:337: PytestUnknownMarkWarning: Unknown pytest.mark.copyright - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
17:29:00     PytestUnknownMarkWarning,
17:29:00 
17:29:00 /Users/osrf/jenkins-agent/workspace/ci_osx/venv/lib/python3.7/site-packages/_pytest/mark/structures.py:337
17:29:00   /Users/osrf/jenkins-agent/workspace/ci_osx/venv/lib/python3.7/site-packages/_pytest/mark/structures.py:337: PytestUnknownMarkWarning: Unknown pytest.mark.linter - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
17:29:00     PytestUnknownMarkWarning,
17:29:00 
17:29:00 /Users/osrf/jenkins-agent/workspace/ci_osx/venv/lib/python3.7/site-packages/_pytest/mark/structures.py:337
17:29:00   /Users/osrf/jenkins-agent/workspace/ci_osx/venv/lib/python3.7/site-packages/_pytest/mark/structures.py:337: PytestUnknownMarkWarning: Unknown pytest.mark.flake8 - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
17:29:00     PytestUnknownMarkWarning,
17:29:00 
17:29:00 /Users/osrf/jenkins-agent/workspace/ci_osx/venv/lib/python3.7/site-packages/_pytest/mark/structures.py:337
17:29:00   /Users/osrf/jenkins-agent/workspace/ci_osx/venv/lib/python3.7/site-packages/_pytest/mark/structures.py:337: PytestUnknownMarkWarning: Unknown pytest.mark.pep257 - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
17:29:00     PytestUnknownMarkWarning,
```

**Comment by dirk-thomas** [comment]
Please see #164.
